mojaloop/values.yaml

```diff
@@ -7113,7 +7113,9 @@ ml-testing-toolkit:
           "payerfspSettlementAccountBalanceAfterFundsIn": "",
           "fundsInPrepareTransferId": "",
           "fundsInPrepareAmount": "",
-          "SIMPLE_ROUTING_MODE_ENABLED": true
+          "SIMPLE_ROUTING_MODE_ENABLED": true,
+          "mobileSimPayerFsp": "pinkbankfsp",
```

Comment:
Please also add `"ON_US_TRANSFERS_ENABLED": false` to make this "config complete".

Reply:
Sure